multus/multus.go

```diff
@@ -294,6 +294,24 @@ func cmdDel(args *skel.CmdArgs, exec invoke.Exec, kubeClient k8s.KubeClient) err
 		return err
 	}
 
+	if args.Netns == "" {
+		return nil
+	}
+	netns, err := ns.GetNS(args.Netns)
+	if err != nil {
+		if err != nil {
+			//  if NetNs is passed down by the CEO, or if it called multiple times
```

Comment:
one q. what is the CEO?

Reply:
Sorry that should Cloud Orchestration Engine